### PR TITLE
Adding LMS verify service to runtime

### PR DESCRIPTION
**Pull request by nquarton**

### CODE REVIEW

**Comment by sree-revoori1** [comment]
Please add the tests to runtime/doc/test-coverage.md

**Comment by nquarton** [comment]
> Please add the tests to runtime/doc/test-coverage.md

Thanks for pointing this out! It has been added now.

File: runtime/src/verify.rs [hunk 1]
```diff
@@ -14,13 +14,16 @@ Abstract:
...
-    Ecc384Signature,
+    Ecc384Signature, LmsResult,
 };
-
-use zerocopy::FromBytes;
+use caliptra_lms_types::{
+    LmotsAlgorithmType, LmotsSignature, LmsAlgorithmType, LmsPublicKey, LmsSignature,
+};
+use zerocopy::AsBytes;
+use zerocopy::{BigEndian, FromBytes, LittleEndian, U32};
```
Comment (leongross) [inline runtime/src/verify.rs:26]: Readability nit:
```suggestion
use zerocopy::{AsBytes, BigEndian, FromBytes, LittleEndian, U32};
```

Comment (nquarton) [reply runtime/src/verify.rs:26]: Thanks, I did not catch this. I'll correct this in the next patch.

**Review by jhand2 — APPROVED** [review]

**Pull request merged**